adodbapi/apibase.py

```diff
@@ -461,20 +465,25 @@ def convert_to_python(variant, func):  # convert DB value into Python value
     return func(variant)  # call the appropriate conversion function
 
 
-class MultiMap(dict):  # builds a dictionary from {(sequence,of,keys) : function}
-    """A dictionary of ado.type : function -- but you can set multiple items by passing a sequence of keys"""
+class MultiMap(Dict[int, Callable[[object], object]]):
+    # builds a dictionary from {(iterable,of,keys) : function}
+    """A dictionary of ado.type : function
+    -- but you can set multiple items by passing an iterable of keys"""
 
     # useful for defining conversion functions for groups of similar data types.
-    def __init__(self, aDict):
-        for k, v in list(aDict.items()):
+    def __init__(self, aDict: Mapping[Iterable[int] | int, Callable[[object], object]]):
+        for k, v in aDict.items():

```

Comment:
`ItemsView` (or in the case of a `dict`, `dict_items`) is already iterable. coercing to a list is redundant.